integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java

```diff
@@ -0,0 +1,151 @@
+              "source=%s | eval f = convert_tz('2021-05-12 11:34:50','-08:00','+09:00') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows("2021-05-13 04:34:50"));
+    }
+
+    @Test
+    public void inRangeNoTZChange() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-12 11:34:50','+09:00','+09:00') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows("2021-05-12 11:34:50"));
+    }
+
+    @Test
+    public void inRangeTwentyFourHourChange() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-12 11:34:50','-12:00','+12:00') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows("2021-05-13 11:34:50"));
+    }
+
+    @Test
+    public void inRangeFifteenMinuteTZ() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-12 13:00:00','+09:30','+05:45') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows("2021-05-12 09:15:00"));
+    }
+
+    @Test
+    public void nullFromFieldUnder() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-30 11:34:50','-17:00','+08:00') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows(new Object[]{null}));
+    }
+
+    @Test
+    public void nullToFieldOver() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-12 11:34:50','-12:00','+15:00') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows(new Object[]{null}));
+    }
+
+    @Test
+    public void nullFromGarbageInput1() throws IOException {
+      JSONObject result =
+          executeQuery(String.format(
+              "source=%s | eval f = convert_tz('2021-05-12 11:34:50','-12:00','test') | fields f",
+              TEST_INDEX_DATE));
+      verifySchema(result,
+          schema("f", null, "datetime"));
+      verifySome(result.getJSONArray("datarows"), rows(new Object[]{null}));
+    }
+
+    @Test
+    public void nullFromGarbageInput2() throws IOException {
```

Comment:
Can the method name describe what's garbage here?

Reply:
Name was made more descriptive.